### DISCUSSION TITLE
playlist sort natural way

### DISCUSSION THREAD

**Discussion by mh-firouzjah** [smplayer-dev/smplayer#327] [Q&A]
hi there. as you may noticed inside the smplayer or the old-forum, playlist does sort items but it sort and consider numbers as string rather than real number orders. so for example it would play the file which it's name starts with 10 sooner than file name starts with 2.
how to fix this problem?
it seems it's a common problem as mentioned here: [stackoverflow.com - string-natural-sort](https://stackoverflow.com/questions/4836710/is-there-a-built-in-function-for-string-natural-sort)

* for now I have written a [python code](https://gist.github.com/mh-firouzjah/24a4f48d401c8271d2605fc8f1dee224) in-order to rename files so they would be in correct order.

**Comment by ElektroStudios** [comment]
Please we can continue the discussion here: https://github.com/smplayer-dev/smplayer/issues/313
